.github/workflows/lint.yml

```
@@ -14,3 +14,30 @@ jobs:
         with:
           python-version: "3.x"
       - uses: pre-commit/action@v3.0.1
+
+  stubtest:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python "3.12"
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          allow-prereleases: true
```

Let's use latest stable (currently 3.13) so we don't need to manually bump:
```suggestion
      - name: Set up Python
        uses: actions/setup-python@v5
        with:
          python-version: "3.x"
```